plugins/org.jboss.reddeer.workbench/src/org/jboss/reddeer/workbench/impl/editor/AbstractEditor.java

```diff
@@ -180,6 +183,44 @@ public ContentAssistant openContentAssistant() {
         return new ContentAssistant(caw.getContentAssistTable());
     }
 
+	public ContentAssistant openContentAssistant(ContentAssistantEnum assistantType) {
```

Comment:
existing openContentAssistant() should also use the new way of opening contentAssistant (via ShellMenu)